pkg/upgrade/upgrade.go

```diff
@@ -255,6 +255,10 @@ func CleanupExistingResource(ctx context.Context,
 	deprecatedFeatureTrackers := []string{dscApplicationsNamespace + "-kserve-temporary-fixes"}
 	multiErr = multierror.Append(multiErr, deleteDeprecatedResources(ctx, cli, dscApplicationsNamespace, deprecatedFeatureTrackers, &featuresv1.FeatureTrackerList{}))
 
+	// Cleanup of deprecated default RoleBinding resources
```

Comment:
i am fine with  the change to not create new association betwee "default" and "anyuid" in any new build, 
I leave review of "upgrade path" to others for comments.